lib/mutant/reporter/hash/printer/env_progress.rb

```diff
@@ -0,0 +1,64 @@
+module Mutant
+  class Reporter
+    class Hash
+      class Printer
+        # Env progress printer
+        class EnvProgress < self
+          delegate(
+            :coverage,
+            :amount_subjects,
+            :amount_mutations,
+            :amount_mutations_alive,
+            :amount_mutations_killed,
+            :runtime,
+            :killtime,
+            :overhead,
+            :env
+          )
+
+          # Run printer
+          #
+          # @return [undefined]
+          #
+          # rubocop:disable AbcSize
+          #
+          # @api private
+          def run
+            {
+              config: visit(Config, env.config),
+              subjects: amount_subjects,
+              mutations: amount_mutations,
+              kills: amount_mutations_killed,
+              alive: amount_mutations_alive,
+              runtime: runtime,
+              killtime: killtime,
+              overhead: overhead_percent.to_f,
+              coverage: coverage_percent.to_f,
+              expected: (env.config.expected_coverage * 100).to_f
```

Comment:
Lets push the float version projection down to the config object instead of duplicating the rational * 100 to_f everywhere.